app/validators/base_validator.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class BaseValidator < ActiveModel::Validator
+  def facility_code_in_weam?(record)
```

Comment:
This one can be `private` too.  Only `validate` (on the subclass) is ever called directly